public/index.html

```diff
@@ -17,6 +17,17 @@
         f.parentNode.insertBefore(j, f)
       })(window, document, 'script', 'dataLayer', 'GTM-PZJ7X57')
     </script>
+    <script>
+      var oldErrorHandler = window.onerror;
```

Comment:
1. use `addEventListener` way to bind the events
2. also, unregister the service worker just before reload (like we're doing in the `retryImport` fn)
3. add some function to remove this listener to the window namespace. e.g. `window.unregisterXXXX = function () { // removeEventListener call here }`
4. call unregister fn from the Router Selector  (in the mounted effect where we calling also `initAnalytics()`)